### PR TITLE
Add a Distribution team to the project governance

### PR DESCRIPTION
**Pull request by jkutner**

### CODE REVIEW

**Review by nebhale — APPROVED** [review]

**Review by ekcasey — APPROVED** [review]

File: TEAMS.md [hunk 1]
```diff
@@ -26,13 +26,21 @@
 
 [@ekcasey][@ekcasey], [@jromero][@jromero], [@sclevine][@sclevine]
 
+## Distribution Team
+
+> The Distribution team is responsible for maintaining tools and services that support the distribution, discovery, and integration of buildpacks.
+
+### Maintainers
+
+TBD
```
Comment (hone) [inline TEAMS.md:35]: Are we gating on solving this or we want to leave this empty?
```suggestion
[@jkutner][@jkutner]
```

File: TEAMS.md [hunk 2]
```diff
@@ -26,13 +26,21 @@
...
+
```
Comment (dfreilich) [inline TEAMS.md:36]: Move below ### Project Contributors

**Review by dfreilich — CHANGES_REQUESTED** [review]
This looks like you are changing all of the Platform Team Contributors to be Distribution Team Contributors, which I don't think is intended.

File: TEAMS.md [hunk 3]
```diff
@@ -26,13 +26,21 @@
...
+> The Distribution team is responsible for maintaining tools and services that support the distribution, discovery, and integration of buildpacks.
+
+### Maintainers
+
+[@jkutner][@jkutner]
+
 ### Project Contributors
 
 [@ameyer-pivotal][@ameyer-pivotal], [@ashwin-venkatesh][@ashwin-venkatesh], [@dfreilich][@dfreilich], [@elbandito][@elbandito], [@matthewmcnew][@matthewmcnew], [@micahyoung][@micahyoung], [@murphybytes][@murphybytes], [@natalieparellano][@natalieparellano], [@simonjjones][@simonjjones], [@yaelharel][@yaelharel], [@zmackie][@zmackie]
```
Comment (jkutner) [inline TEAMS.md:39]:
```suggestion
[@elbandito][@elbandito]
```

File: TEAMS.md (hunk 2 above)
Comment (jkutner) [reply TEAMS.md]: Fixed in 68f7b86

Comment (dfreilich) [reply TEAMS.md]: It still looks like all of the Platform team Contributors are being deleted here.

File: TEAMS.md [hunk 4]
```diff
@@ -26,13 +26,21 @@
 
 [@ekcasey][@ekcasey], [@jromero][@jromero], [@sclevine][@sclevine]
```
Comment (jkutner) [inline TEAMS.md:28]:
```suggestion

### Project Contributors 

- [@ameyer-pivotal][@ameyer-pivotal], [@ashwin-venkatesh][@ashwin-venkatesh], [@dfreilich][@dfreilich], [@elbandito][@elbandito], [@matthewmcnew][@matthewmcnew], [@micahyoung][@micahyoung], [@murphybytes][@murphybytes], [@natalieparellano][@natalieparellano], [@simonjjones][@simonjjones], [@yaelharel][@yaelharel], [@zmackie][@zmackie]
```

File: TEAMS.md [hunk 5]
```diff
@@ -26,13 +26,24 @@
...
 
+### Project Contributors 
+
+- [@ameyer-pivotal][@ameyer-pivotal], [@ashwin-venkatesh][@ashwin-venkatesh], [@dfreilich][@dfreilich], [@elbandito][@elbandito], [@matthewmcnew][@matthewmcnew], [@micahyoung][@micahyoung], [@murphybytes][@murphybytes], [@natalieparellano][@natalieparellano], [@simonjjones][@simonjjones], [@yaelharel][@yaelharel], [@zmackie][@zmackie]
```
Comment (jkutner) [inline TEAMS.md:31]:
```suggestion
[@ameyer-pivotal][@ameyer-pivotal], [@ashwin-venkatesh][@ashwin-venkatesh], [@dfreilich][@dfreilich], [@elbandito][@elbandito], [@matthewmcnew][@matthewmcnew], [@micahyoung][@micahyoung], [@murphybytes][@murphybytes], [@natalieparellano][@natalieparellano], [@simonjjones][@simonjjones], [@yaelharel][@yaelharel], [@zmackie][@zmackie]
```

File: TEAMS.md (hunk 2 above)
Comment (jkutner) [reply TEAMS.md]: ah, i see what you mean. i think have it now with 4f434f4

**Review by dfreilich — APPROVED** [review]

**Review by jromero — APPROVED** [review]

**Review by jromero — APPROVED** [review]

**Pull request merged**